test/cypress/integration/widgets/plugins_catalog_spec.ts

```diff
@@ -106,4 +106,26 @@ describe('Plugins Catalog widget', () => {
             '-'
         );
     });
+
+    it('should upload all plugins', () => {
+        // eslint-disable-next-line security/detect-non-literal-regexp
+        cy.intercept('POST', new RegExp(`console/plugins/upload.*title=AWS`)).as('awsPluginUpload');
+        cy.contains('Upload all plugins').click().should('be.disabled');
+        cy.get('.pluginsCatalogWidget tr button').should('be.disabled');
+        // eslint-disable-next-line security/detect-non-literal-regexp
+        cy.intercept('POST', new RegExp(`console/plugins/upload.*title=Ansible`)).as('ansiblePluginUpload');
+        cy.wait('@awsPluginUpload', { responseTimeout: uploadPluginTimeout });
+        cy.contains('AWS successfully uploaded');
+        cy.contains('.pluginsCatalogWidget tr', 'AWS').find('button.loading').should('not.exist');
```

Comment:
Optional:

Maybe for easier reading we can have:
1. Both intercepts at the top
2. Some spacing between setup, test and assertion blocks
WDYT?

```suggestion
        // eslint-disable-next-line security/detect-non-literal-regexp
        cy.intercept('POST', new RegExp(`console/plugins/upload.*title=AWS`)).as('awsPluginUpload');
        // eslint-disable-next-line security/detect-non-literal-regexp
        cy.intercept('POST', new RegExp(`console/plugins/upload.*title=Ansible`)).as('ansiblePluginUpload');
        
        cy.contains('Upload all plugins').click().should('be.disabled');
        
        cy.get('.pluginsCatalogWidget tr button').should('be.disabled');
        cy.wait('@awsPluginUpload', { responseTimeout: uploadPluginTimeout });
        cy.contains('AWS successfully uploaded');
        cy.contains('.pluginsCatalogWidget tr', 'AWS').find('button.loading').should('not.exist');
```

Reply:
Actually my intension here was to test that the second plugin upload starts after the first one, not in parallel to the first one. Moving the second intercept to the top would make the test to pass regardless of requests timing. I consider it important as launching all plugin uploads at once was causing the backend to freeze for a while making the application unresponsive.

I added empty lines as suggested though. Also added a comment.

Reply:
OK, got it, sounds reasonable to have it in that order then. Thanks for adding the comment about it.